neutone_sdk/utils.py

```diff
@@ -160,13 +161,14 @@ def save_neutone_model(
         loaded_model.set_daw_sample_rate_and_buffer_size(48000, 512)
         loaded_model.reset()
         loaded_model.is_resampling()
-        log.info(
-            f"Buffering delay reported to the DAW for 48000 Hz sampling rate and 512 buffer size: "
-            f"{loaded_model.calc_buffering_delay_samples()}"
+        log.info("Running latency benchmark...")
+        benchmark_latency_(
+            str(root_dir / "model.nm"),
+            buffer_size=[128, 256, 512, 1024, 2048],
+            sample_rate=[48000],
         )
         log.info(
-            f"Model delay reported to the DAW for 48000 Hz sampling rate and 512 buffer size: "
-            f"{loaded_model.calc_model_delay_samples()}"
+            "We recommend running additional latency/speed benchmarks to get a better idea of model performance."
```

Comment:
Here we should mention how to do this if we add the additional flags to the `save_neutone_model` method